seacrowd/sea_datasets/sentiment_taglish_product_review/sentiment_taglish_product_review.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""\
+This dataset contains 10,510 examples of product and service reviews in Taglish
+from Google Maps Reviews and Shopee Philippines. Reviews are manually labeled by
+three human annotators according to four sentiment classes: Positive, Negative, Neutral, and Mixed.
+"""
+import csv
+from pathlib import Path
+from typing import Dict, Generator, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# no citation found for this dataset
+_CITATION = ""
+_DATASETNAME = "sentiment_taglish_product_review"
+_DESCRIPTION = """\
+Sentiment-Annotated Taglish Product and Service Reviews (SentiTaglish: Products and
+Services) is a gold standard, sentiment-annotated corpus for the Tagalog-English
+language pair. It contains 10,510 product and service reviews which were manually
+labeled by three human annotators according to four sentiment classes:
+Positive, Negative, Neutral, and Mixed.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/ccosme/SentiTaglishProductsAndServices"
+
+_LANGUAGES = ["tgl", "eng"]
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
```

Comment:
I double-checked #720 and the HF dataset and it seems like it should just be CC-by-4.0 🤔 

```suggestion
_LICENSE = Licenses.CC_BY_4_0.value
```